frontend/language/src/nb.json

```diff
@@ -1791,13 +1792,13 @@
   "ux_editor.upload_file_error_too_large": "Kunne ikke laste opp filen. Den er for stor.",
   "ux_editor.url_label": "Lenke",
   "ux_editor.warning": "Advarsel",
-  "validation_errors.length": "Antall tillatte tegn er {{0}}",
-  "validation_errors.max": "Største gyldig verdi er {{0}}",
-  "validation_errors.maxLength": "Bruk {{0}} eller færre tegn",
-  "validation_errors.min": "Minste gyldig verdi er {{0}}",
-  "validation_errors.minLength": "Bruk {{0}} eller flere tegn",
-  "validation_errors.numbers_only": "Kun sifre er gyldige tegn",
-  "validation_errors.pattern": "Feil format eller verdi",
-  "validation_errors.required": "Feltet må fylles ut",
-  "validation_errors.value_as_url": "Ugyldig lenke"
+  "validation_errors.length": "Antall tillatte tegn er {{0}}.",
+  "validation_errors.max": "Største gyldig verdi er {{0}}.",
+  "validation_errors.maxLength": "Bruk {{0}} eller færre tegn.",
+  "validation_errors.min": "Minste gyldig verdi er {{0}}.",
+  "validation_errors.minLength": "Bruk {{0}} eller flere tegn.",
```

Comment:
Maybe all these texts can use some more descriptive variables instead of `0` as well 🙈